types/office-js-preview/index.d.ts

```diff
@@ -42994,6 +43076,16 @@ declare namespace Excel {
          * [Api set: ExcelApi 1.9]
          */
         getRanges(): Excel.RangeAreas;
+        /**
+         * Set the ranges that the conditonal format rule is applied to.
```

Comment:
Does this also apply the format to those ranges?

Reply:
Good question. I think yes, but I will confirm with Eng/PM. Are you ok with approving this PR so that I can merge it, with the expectation that I'll update the description once I get confirmation? Or should I wait to merge?